mimic/rest/auth_api.py

```diff
@@ -124,6 +124,22 @@ def get_username(self, request, tenant_id):
         session = self.core.sessions.session_for_tenant_id(tenant_id)
         return json.dumps(dict(user=dict(id=session.username)))
 
+    @app.route('/v2.0/users/<string:user_id>/OS-KSADM/credentials/RAX-KSKEY:apiKeyCredentials',
+               methods=['GET'])
+    def rax_kskey_apikeycredentials(self, request, user_id):
+        """
+        Support, such as it is, for the apiKeysCredentials call.
+        """
+        if user_id in self.core.sessions._userid_to_session:
+            username = self.core.sessions._userid_to_session[user_id].username.decode('ascii')
+            apikey = '7fc56270e7a70fa81a5935b72eacbe29'  # echo -n A | md5sum
+            return json.dumps({'RAX-KSKEY:apiKeyCredentials': {'username': username,
+                               'apiKey': apikey}})
+        else:
+            request.setResponseCode(404)
+            return json.dumps({'itemNotFound':
+                              {'code': 404, 'message': 'User ' + user_id + ' not found'}})
+
```

Comment:
Meh, this is just a stub code so that Agent-Install page doesn't get a 404. I'm actually not trying to make this work 100% like production. I say it's good enough for now because my real goal is just getting the e2e test working.